File: courses/C# 2008 - Impacta/C# 2008 - M2/Instrutor/CSModII20090918/Laboratórios/Capitulo01_Laboratorio01/Capitulo01_Laboratorio01/Cap01_Lab01Form.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

using System.Diagnostics;

namespace Capitulo01_Laboratorio01
{
    public partial class Cap01_Lab01Form : Form
    {
        public Cap01_Lab01Form()
        {
            InitializeComponent();
        }

        private void calculadoraButton_Click(object sender, EventArgs e)
        {
            Process[] processos = Process.GetProcessesByName("calc");

            if (processos.Length == 0)
            {
                Process.Start("calc.exe");
            }
            else
            {
                MessageBox.Show("A calculadora já está aberta", "Alerta",
                    MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
            }
        }
    }
}

File: courses/C# 2008 - Impacta/C# 2008 - M2/Instrutor/CSModII20090918/VB.NET2008 Mod2 Hanna/VBNet II/Módulo II/Laboratórios/Capitulo01_Laboratorio01/Capitulo01_Laboratorio01/Cap01_Lab01Form.vb
Public Class Cap01_Lab01Form

    Private Sub calculadoraButton_Click(ByVal sender As System.Object, _
                            ByVal e As System.EventArgs) Handles calculadoraButton.Click

        Dim processos() As Process = Process.GetProcessesByName("calc")

        If processos.Length = 0 Then
            Process.Start("calc.exe")
        Else
            MessageBox.Show("A calculadora já está aberta", "Alerta", _
                    MessageBoxButtons.OK, MessageBoxIcon.Exclamation)
        End If
    End Sub

End Class
